packages/devtools_app/lib/src/shared/server/_analytics_api.dart

```diff
@@ -56,6 +56,32 @@ Future<bool> setAnalyticsEnabled([bool value = true]) async {
   return false;
 }
 
+/// Fetch the consent message for package:unified_analytics.
+Future<String?> getConsentMessage() async {
+  String? consentMessage;
+  if (isDevToolsServerAvailable) {
+    final resp = await request(apiGetConsentMessage);
```

Comment:
nit: rename `apiGetConsentMessage` to `apiAnalyticsConsentMessage`? to make it clear this is for analytics